modules/appeals_api/app/services/appeals_api/pdf_construction/notice_of_disagreement/v1/structure.rb

```diff
@@ -179,6 +168,34 @@ def additional_pages?
           form_data.hearing_type_preference.present? ||
             form_data.contestable_issues.count > 5
         end
+
+        # rubocop:disable Metrics/MethodLength
+        def insert_issues_into_text_boxes(pdf, text_opts)
```

Comment:
The way this is written it seems like if both the issue text & disagreementReason are long, the two bounding boxes will overlap each other before they start autosizing.

I think we should detect if an issue has a disagreement reason and change the bounding boxes between the reason & disagreement so there isn't a possibility of overlap.

We'd need to change the `ypos` & `height` of each depending on whether `disagreementReason` is present/blank.

Let's also double check the documentation in case there's character limits (maxLength) there and adjust them if need be (unless that's already different ticket).

Reply:
Discussed on zoom.